deny.toml

```diff
@@ -58,6 +59,7 @@ skip = [
   { name = "cargo_metadata" },      # Older version used by ply-rs. It's small, and it's build-time only!
   { name = "core-foundation" },     # Currently, e.g. `webbrowser` and `winit` use different versions.
   { name = "core-graphics-types" }, # wgpu requires 0.2 while winit is still on 0.1
+  { name = "event-listener" },      # remove after https://github.com/lancedb/lance/pull/4834 closes
```

Comment:
again, by using  `#TODO(#issue)` format, it gets picked up by our zombie todo script